schema/proto3/port.proto

```diff
@@ -23,18 +23,24 @@ option java_outer_classname = "Port";
 
 import "common.proto";
 
+enum MessageType { 
+    DELTA = 0; // the default type
+    FULL = 1;
+}
+
 message PortConfiguration {
-    uint32 version = 1;
-    
-    NetworkType network_type = 2;
-    string project_id = 3;
-    string network_id = 4;
-    string id = 5;
-    string name = 6;
-    string network_ns = 7;
-    string mac_address = 8;
-    string veth_name = 9;
-    bool admin_state_up = 10;
+    uint32 format_version = 1;
+    uint32 content_version = 2;
+    MessageType message_type = 3;
+
+    string id = 4;
+    NetworkType network_type = 5;
+    string project_id = 6;
+    string subnet_id = 7;
+    string name = 8;
+    string network_ns = 9;
```

Comment:
Is network_ns used for Mizar only? If so, we could move it to the end.

Reply:
we will likely use it for k8s integration based on our previous design, since all the containers endpoints are under some network namespace.